Testscripts/Linux/utils.sh

```diff
@@ -3730,4 +3730,35 @@ function get_bootconfig_path() {
 		config_path="/usr/boot/config-$(uname -r)"
 	fi
 	echo "$config_path"
+}
+
+function Check_lsvmbus()
```

Comment:
This is a good PR. But we do a good practice in the common function writing better. Can you move your comment to the function's description, and add a new comment for return value? This helps other function callers access easy.